pages/server/development/rdb.mdx

```diff
@@ -1,88 +1,129 @@
 import {
+  DiscordEmbedDescription,
   DiscordMessages,
   DiscordMessage,
   DiscordEmbed,
   DiscordEmbedField,
   DiscordEmbedFields,
+  DiscordEmbedFooter,
 } from "@skyra/discord-components-react";
 
 # What is RedDiscordBot? (RDB)
 
 Red discord bot is a multi moduler discord bot. It means that it just has some base of commands. And you can add more commands by adding a thing called cogs (we can say plugins in mc fashion 😎) it had over 550 cogs and has many commands by default.
 
-# Creating a server
+# Creating the server
 
-To create a server, go to DBH's discord server and type the commands below
+To create a server, we suppose that you've already [created and linked an account into the panel](/getting-started).
+
+In that case, go to DBH server and run this command:
```

Comment:
This should be grammatically correct, which is how it was before `go to DBH’s Discord server and run this command:`
Or `go to the DBH Discord server and run this command:`